src/software/world/ball.cpp

```diff
@@ -21,22 +20,17 @@ BallState Ball::currentState() const
     return current_state_;
 }
 
-const std::shared_ptr<BallModel> &Ball::ballModel() const
-{
-    return ball_model_;
-}
-
-void Ball::updateState(const BallState &new_state, const Timestamp &new_timestamp)
+void Ball::updateState(const BallState &new_state, const Timestamp &new_timestamp,
+                       const Vector &new_acceleration)
 {
     if (new_timestamp < timestamp())
     {
         throw std::invalid_argument(
             "Error: Trying to update ball state using a state older then the current state");
```

Comment:
I think you meant "than" here

Reply:
You're finding a lot of spelling mistakes in our existing code. It'd be great if you could put  up a PR to fix a bunch of them at once. (I'll fix this one though)